setup.py

```diff
@@ -38,8 +38,8 @@
         long_description_content_type="text/markdown",
         url=f"https://github.com/ansys/{package_name}",
         license="MIT",
-        python_requires=">=3.9",
-        install_requires=["grpcio~=1.49", "protobuf>=3.19,<6", "ansys-api-dbu==0.3.5"],
```

Comment:
why grpcio package version is changed to an older one here?

Reply:
We have not changed it - we are allowing older versions if needed. The problem is that ``ansys-tools-protoc-helper==0.5.0`` requires Python 3.9 to 3.11 and grpcio 1.49.x. However, when uploading the package to the condaforge channel, this grpcio version is not available, so we have to allow build to take place with ``ansys-tools-protoc-helper==0.4.0`` as welll, which requires Python 3.7 to 3.10.

However, on our pipelines, we are using Python 3.11 and this will force the usage of  ``ansys-tools-protoc-helper==0.5.0`` and also grpcio 1.49.x. Consequently, we are not impacted. This change was to allow build on condaforge as well, that's it =)

Reply:
makes sense, thanks for explanation.

Reply:
Anytime!